### DISCUSSION TITLE
Need help understanding my graphs!

### DISCUSSION THREAD

**Discussion by Kristufar** [gephi/gephi#2746] [Q&A]
Hi, 

I’m new to Gephi/network analysis and I’m enjoying learning it, but something has stumped me. Can I get some feedback please?   

Basically I’ve produced two network graphs, both of which have as nodes the articles from the same academic journal. However, one of the graphs uses data about the citations _of_ the articles for the edges, while the other uses data about citations _by_ for the edges.   The thing is the two graphs are visually very different despite me using the same algorithm (ForceAtlas2), and I can’t see why they should be different because citations of and citations by are essentially the same thing, just in reverse!

Please help me get my head around this!

<img width="361" alt="image" src="https://user-images.githubusercontent.com/123576493/214616252-72ad174f-eccd-4752-a074-369d1aba5421.png">

<img width="425" alt="image" src="https://user-images.githubusercontent.com/123576493/214616313-a9dace22-8fad-4025-ad78-38ea4ad9c754.png">

**Comment by seinecle** [comment]
Hi, 

You will get help and useful feedback on the Facebook group for Gephi:
https://facebook.com/groups/gephi
